File: core/src/com/mygdx/game/model/weapon/Pistol.kt
package com.mygdx.game.model.weapon

import com.mygdx.game.model.projectile.ProjectileType
import com.mygdx.game.settings.PISTOL_BULLETS_IN_CHAMBER
import com.mygdx.game.settings.PISTOL_RELOAD_TIME

class Pistol : Weapon(PISTOL_RELOAD_TIME, PISTOL_BULLETS_IN_CHAMBER, ProjectileType.PISTOL)

File: server/models/weapons/Pistol.js
const constants = require('../../settings/constants');
const Weapon = require('./Weapon');
const ProjectileType = require('../projectiles/ProjectileType');

class Pistol extends Weapon {
    constructor() {
        super(constants.PISTOL_RELOAD_TIME, constants.PISTOL_BULLETS_IN_CHAMBER, constants.PISTOL_MAGAZINE_RELOAD_TIME,
            ProjectileType.PISTOL)
    }
}

module.exports = Pistol;